pages/common/nixos-rebuild.md

```diff
@@ -0,0 +1,31 @@
+# nixos-rebuild
+
+> Reconfigure a NixOS machine.
+
+- Build and switch to the new configuration, making it the boot default:
```

Comment:
I see this example has basically the same description as the 5th one. What's the difference? I think one of them (if not both) should be reworded to be clearer.